CHANGELOG.md

```diff
@@ -8,10 +8,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## UNRELEASED
 
 ### Added
+
+- Use the module's name when the input has only one module (#1260)
```

Comment:
iiuc, something like this would be a clearer statement of the impact of this change:

```suggestion
- When an input file only one module, it will be inferred as the main module  (#1260)
```